### PR TITLE
Add base styles

### PR DESCRIPTION
**Pull request by brunvez**

### CODE REVIEW

File: spec/helpers/exception_hunter/errors_helper_spec.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# describe ErrorsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+module ExceptionHunter
+  RSpec.describe ErrorsHelper, type: :helper do
+    pending "add some examples to (or delete) #{__FILE__}"
```
Comment (t-romani) [inline spec/helpers/exception_hunter/errors_helper_spec.rb:15]: 👀

File: spec/views/exception_hunter/errors/index.html.erb_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "errors/index.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (t-romani) [inline spec/views/exception_hunter/errors/index.html.erb_spec.rb:4]: 👀

File: app/helpers/exception_hunter/errors_helper.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+module ExceptionHunter
+  module ErrorsHelper
```
Comment (t-romani) [inline app/helpers/exception_hunter/errors_helper.rb:2]: 🤔 what's this for?

Comment (brunvez) [reply app/helpers/exception_hunter/errors_helper.rb]: Helpers get included in every view. They provide helper methods to format strings and the likes 

https://www.rubyguides.com/2020/01/rails-helpers/

**Pull request merged**